src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Enums are classes and should follow the conventions for classes. Instances of an enum are constants and should follow the conventions for constants ->  uppercase letters

more info - https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names